src/domain/common/whiteboard/whiteboard.service.ts

```diff
@@ -222,6 +224,11 @@ export class WhiteboardService {
 
     this.eventEmitter.emit(WHITEBOARD_CONTENT_UPDATE, savedWhiteboard.id);
 
+    this.subscriptionPublishService.publishWhiteboardSaved(
+      whiteboard.id,
+      new Date(Date.now())
```

Comment:
The behavior you described in the PR description is misleading to the end user and can easily be avoided.
Persisting a large whiteboard is not ideal, so try to update only the update date, or if nothing else works, persist the whole whiteboard.
We don't want users to be afraid of using the feature, because the last save doesn't change consistently.

Reply:
good good, just updating the `updateDate` manually works, so we have that weird behavior fixed now. I'll change the description of the PR, thanks.